### PR TITLE
Added timeout configs and variables to aws_eks_cluster resource

### PR DESCRIPTION
**Pull request by RGPosadas**
# PR o'clock

## Description

Added timeout configs to the `aws_eks_cluster` resource.

### Test Result
I set the timeouts to 1m to test for failure (could not test timeout for more than default time since it is unpredictable when it would happen). Here are the results:
```
1 error(s) occurred:
TestEKS 2018-10-02T12:32:30-04:00 command.go:100: 
TestEKS 2018-10-02T12:32:30-04:00 command.go:100: * module.test_eks_module.module.eks.aws_eks_cluster.this: 1 error(s) occurred:
TestEKS 2018-10-02T12:32:30-04:00 command.go:100: 
TestEKS 2018-10-02T12:32:30-04:00 command.go:100: * aws_eks_cluster.this: timeout while waiting for state to become 'ACTIVE' (last state: 'CREATING', timeout: 1m0s)
```

### `README.md` Changes
I tried to update `README.md` using `terraform-docs` and the given lines of code, but that ended up erasing the whole file for me. So I added in the new variables manually.

### Checklist

- [ ] `terraform fmt` and `terraform validate` both work from the root and `examples/eks_test_fixture` directories (look in CI for an example)
- [ ] Tests for the changes have been added and passing (for bug fixes/features)
- [x] Test results are pasted in this PR (in lieu of CI)
- [ ] Docs have been updated using `terraform-docs` per `README.md` instructions
- [x] I've added my change to CHANGELOG.md
- [ ] Any breaking changes are highlighted above

### CODE REVIEW

**Comment by laverya** [comment]
I think it might be best if you merge master into this branch before going much farther - it looks like your branch is off of https://github.com/terraform-aws-modules/terraform-aws-eks/commit/b6f6a823524c9e887a1c0f5a7bf441c328781e02, and there have been a number of changelog additions since then. (which is causing the merge conflict)

I like the change though! I've certainly run into the create timeout before.

Re: `README.md`, what version of terraform-docs (and go, and the os...) are you running? I just tried generating the docs again with this set of commands:
```
go get -u github.com/segmentio/terraform-docs
terraform-docs md ./ | cat -s | tail -r | tail -n +2 | tail -r > README.md
```
and had no differences.

File: variables.tf [hunk 1]
```diff
@@ -123,3 +123,13 @@ variable "kubeconfig_name" {
   description = "Override the default name used for items kubeconfig."
   default     = ""
 }
+
+variable "aws_eks_cluster_create_timeout" {
```
Comment (mmcaya) [inline variables.tf]: For consistency with other `cluster_*` related variables and readability (e.g. `cluster_version`, should these be named `cluster_create_timeout` and `cluster_delete_timeout`?

Comment (RGPosadas) [reply variables.tf]: Sure, I'll change it

**Comment by RGPosadas** [comment]
Hey @laverya ! 
I synced up my fork with master and resolved the merge conflict.

**Comment by mmcaya** [comment]
Thanks @RGPosadas, README.md should also be updated to reflect the new var names.

**Comment by RGPosadas** [comment]
I updated `README.md`!

Also re: `README.md`, had to change `tail -r` to `tac` and it worked.

**Comment by max-rocket-internet** [comment]
Thanks for your first contribution @RGPosadas!

I just fixed the travis error in master, could you rebase so we can get it to pass? Thanks!

**Pull request merged**